etcdserver/api/membership/cluster.go

```diff
@@ -691,6 +702,39 @@ func clusterVersionFromBackend(lg *zap.Logger, be backend.Backend) *semver.Versi
 	return semver.Must(semver.NewVersion(string(vals[0])))
 }
 
+func downgradeInfoFromBackend(lg *zap.Logger, be backend.Backend) *DowngradeInfo {
```

Comment:
This is an unexported func without caller.

Can it be removed ?

Reply:
The caller of this func will be added in following PRs. I want to put the getter and setter of downgradeInfo in one PR.